### PR TITLE
Additional render options

### PR DESCRIPTION
**Pull request by jdent02**
Bounce counts, volume settings, and lighting sampler

Note: export only option is partially complete so it isn't exposed in the ui yet.

### CODE REVIEW

File: properties/scene.py [hunk 1]
```diff
@@ -147,14 +153,20 @@ def register(cls):
                                                     min=1,
                                                     max=999999)
 
+        cls.light_sampler = bpy.props.EnumProperty(name="Light Sampler",
+                                                  description="The method used for sampling lights",
+                                                  items=[('cdf', 'CDF', 'CDF'),
+                                                        ('lighttree', 'Lights Tree', 'Light Tree')],
```
Comment (dictoon) [inline properties/scene.py]: `Lights Tree` -> `Light Tree`

File: properties/scene.py [hunk 2]
```diff
@@ -198,6 +210,34 @@ def register(cls):
                                                 min=0,
                                                 max=512)
 
+        cls.use_separate_bounces = bpy.props.BoolProperty(name="Use Individual Bounce Limits",
+                                                          description="Use individual limits for different ray types",
+                                                          default=False)
+
+        cls.max_diffuse_bounces = bpy.props.IntProperty(name="Max Diffuse Bounces",
+                                                        description="Maximum diffuse length: 0 = Unlimited",
```
Comment (dictoon) [inline properties/scene.py]: This would be more accurate: `Maximum total number of diffuse bounces: 0 = unlimited` (same for the other bounce types).

**Comment by jdent02** [comment]
Done....

**Comment by dictoon** [comment]
Merged, thank you!

**Pull request merged**